src/forms/ErrorList.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+
+export const ERROR_CLASSNAME = 'text--error text--small';
+
+const ErrorList = (props) => {
+	const {
+		errorId,
+		errors,
+		...other
+	} = props;
+
+	const errorList = Array.isArray(errors) ? errors : [errors];
+
+	if (errorId) {
+		other['id'] = errorId;
+	}
+
+	return (
+		<ul
+			aria-live="assertive"
```

Comment:
Would adding `aria-role="alert"` be too severe?

Reply:
we could add `role="alert"` to the list items, but it would be redundant with the surrounding `assertive` live region